### PR TITLE
Update BZ details in OWNERS file

### PR DESCRIPTION
**Pull request by yboaron**
Adding BZ details to OWNERS file is ART requirement.

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -8,3 +8,7 @@ reviewers:
   - AlonaKaplan
   - eranco74
   - tsorya
+
+# Bugzilla info;
+component: Installer
```
Comment (phoracek) [inline OWNERS:13]: This is the last place I would be opening bug on regarding kubernetes-nmstate day-2 network configuration... Can't we get a new subcomponent under Networking instead?

Comment (yboaron) [reply OWNERS:13]: @phoracek , Well, I just copy-paste BZ details from one of our bugs.
I guess we can change it anytime, if you have another suggestion - I'm open.

Comment (phoracek) [reply OWNERS:13]: @celebdor could we get a new subcomponent under Networking? It would be clearer where to open bugs for users.

Comment (celebdor) [reply OWNERS:13]: yes. I'll request it now

Comment (celebdor) [reply OWNERS:13]: In the meantime, we should probably use Networking, not Installer

**Pull request merged**